external_deps/build.sh

```diff
@@ -654,19 +654,19 @@ common_setup() {
 		export CMAKE_GENERATOR="MSYS Makefiles"
 		CROSS= # Doing this because there is no ${CROSS}strip
 	fi
+	MSVC_SHARED=(--enable-shared --disable-static)
```

Comment:
The MSVC_SHARED change reverses the values on a bunch of platforms, please undo

Reply:
Oh my bad, my eyes really get tricked…

Reply:
Fixed.